packages/protocol/.eslintignore

```diff
@@ -2,3 +2,5 @@ node_modules
 artifacts
 cache
 coverage
+doc
+docs
```

Comment:
I think we want docs automatically updated and posted to github

Reply:
Ah, I can remove it and try again. It was preventing me from committing since there are apparently issues in some doc/docs files, I haven't been able to push without the --no-verify